teampost.md

```diff
@@ -0,0 +1,28 @@
+Are you _tired_ of sifting through notifications? Clicking through so many browser tabs, getting notified so often it makes your computer's memory explode?? There's got to be a better way!
+
+# Introducing [Octoterm](https://github.com/dctucker/octoterm/blob/master/README.md)
```

Comment:
I think due to the way Team does its formatting, you might want a title to be the top line. Maybe something like:

## Octoterm - GitHub notifications in your terminal.

...and then change this "Introducing..." hook line, into an H2. Just a thought, you can play around with it until it looks right. I do think there should be a clear Team post title, with some mention of what the tool does though :+1: